examples/guides/testing/writing-tests-using-data-providers/hello-service/README.md

```diff
@@ -0,0 +1,32 @@
+# Sending-a-simple-message-to-a-service
```

Comment:
I think we don't need a separate hello-service directory. We can simply use the writing-tests-using-data-providers directory since we won't be having multiple examples in this. WDYT?

Reply:
Removed the additional directory.